crates/stackable-operator/CHANGELOG.md

```diff
@@ -11,8 +11,14 @@ All notable changes to this project will be documented in this file.
 - Add functionality to convert LogLevel to an OPA log level ([#798]).
 - BREAKING: Add labels to listener volume builder. `PodBuilder::add_listener_volume_by_listener_class`, `PodBuilder::add_listener_volume_by_listener_name` and `ListenerOperatorVolumeSourceBuilder::new` now require you to pass the labels for the created volumes ([#799]).
 
+### Fixed
+
+- Processing of corrupted log events fixed; If errors occur, the error
+  messages are added to the log event ([#802]).
+
```

Comment:
This needs to be moved to the "Unreleased" section of the changelog.

Reply:
fixed in a81b1312a6a7e62b140d7c07a6c6abdb7f9dcd94